sw/device/silicon_creator/lib/rescue/rescue_null.c

```diff
@@ -0,0 +1,7 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+
+const uint32_t rescue_type = 0;
```

Comment:
OT prefers the use of enums for constants: https://opentitan.org/book/doc/contributing/style_guides/c_cpp_coding_style.html#constants-and-preprocessor-macros